conf/nginx/sites-available/off

```diff
@@ -161,16 +159,21 @@ server {
 
 	location ~ ^/images/products/ {
 		add_header Link "<http://creativecommons.org/licenses/by-sa/3.0/>; rel='license'; title='CC-BY-SA 3.0'";
+		add_header Access-Control-Allow-Origin *;
+	        add_header Access-Control-Allow-Methods 'GET, POST, OPTIONS';
+		add_header Access-Control-Allow-Headers 'DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range,If-None-Match';
+		add_header Access-Control-Expose-Headers 'Content-Length,Content-Range';
 	}
 
+        if ($http_referer ~* (jobothoniel.com) ) { return 403; } # blocked since 2021-07-13
 
 	# the app requests /1.json to get the product count...
 	# the commented code below is to serve a static copy 
 	# if there is a spike of installs
-        #location ~ ^/1.json$ {
-        #        root /srv/off/html/static;
-        #        try_files /1.json = 404;
-        #}
+        location ~ ^/1.json$ {
```

Comment:
Shouldn't it be commented if it's only used "if there is a spike of installs". Or at least comment when it is activated and why.

Reply:
Commented.